### PR TITLE
Update twelve-days to match specification

### LINKED ISSUE #858: Twelve days out of sync with specification
The problem specification for Twelve Days was changed to correct a punctuation issue.

https://github.com/exercism/problem-specifications/blob/84aa47979df7f526bdcb777cacc2c2cc9647dcaa/exercises/twelve-days/description.md

### PR DESCRIPTION
**Pull request by PatrickMcSweeny**
Closes #858.

### CODE REVIEW

File: exercises/twelve-days/song.txt [hunk 1]
```diff
@@ -1,23 +1,23 @@
-On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree.
```
Comment (Insti) [inline exercises/twelve-days/song.txt]: What is this file used for?

**Comment by Insti** [comment]
Awesome, thanks!
Do you want to also have a go at [regenerating the tests](https://github.com/exercism/ruby#regenerating-a-test-suite)?

File: exercises/twelve-days/song.txt (hunk 1 above)
Comment (Insti) [reply exercises/twelve-days/song.txt]: > What is this file used for?

It's used by the test file.

**Comment by PatrickMcSweeny** [comment]
@Insti Done. Incidentally before I saw that you updated your comment I tried regenerating the tests and it looks like several of them changed. I could put that into a new pull request if you like.

**Comment by Insti** [comment]
>  I could put that into a new pull request if you like.

That would be great thanks!

**Comment by Insti** [comment]
Thanks @PatrickMcSweeny :heart:

**Pull request merged**